src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -377,38 +378,313 @@ internal PhysicalLocationVersionOne CreatePhysicalLocation(Region v2Region)
             {
                 physicalLocation = new PhysicalLocationVersionOne
                 {
-                    Region = CreateRegion(v2Region)
+                    Region = CreateRegion(v2Region, null)
```

Comment:
I suggest you name the `null` argument value, `uri: null`. I tripped over this because at first I thought the argument was an encoding.

Reply:
Good call, fixed